src/postgres/tree/PostgresServerTreeItem.ts

```diff
@@ -0,0 +1,58 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import PostgreSQLManagementClient from 'azure-arm-postgresql';
+import { DatabaseListResult, Server } from 'azure-arm-postgresql/lib/models';
+import { Databases } from 'azure-arm-postgresql/lib/operations';
+import * as vscode from 'vscode';
+import { AzureParentTreeItem, AzureTreeItem, ISubscriptionContext } from 'vscode-azureextensionui';
+import { getThemeAgnosticIconPath } from '../../constants';
+import { azureUtils } from '../../utils/azureUtils';
+import { PostgresDatabaseTreeItem } from './PostgresDatabaseTreeItem';
+import { PostgresSchemaTreeItem } from './PostgresSchemaTreeItem';
+import { PostgresTableTreeItem } from './PostgresTableTreeItem';
+
+export class PostgresServerTreeItem extends AzureParentTreeItem<ISubscriptionContext> {
+    public static contextValue: string = "postgresServer";
+    public readonly contextValue: string = PostgresServerTreeItem.contextValue;
+    public readonly childTypeLabel: string = "Database";
+    public readonly client: PostgreSQLManagementClient;
```

Comment:
To continue the previous thread, I still don't want you to cache `client` here in this class. This is the main reason from before:

> I try to avoid caching things like clients because they deal with credentials. I don't know how it works under the covers, but theoretically there could be a refresh token that only gets updated when you create a new client and thus it could expire if you keep an old client around.

To expand on that, consider this possibility:
1. User expands their subscription tree item, causing us to list pg servers and create the client
1. User walks away for a day
1. User comes back and expands a pg server tree item

It's still using the client you created a day ago. I'm okay passing the client around if it's used immediately, but tree expansion does not fit under that case. It's easy enough to just create a new client below and actually results in